### PR TITLE
New example for a data playground Terraform setup

### PR DESCRIPTION
**Pull request by aymanfarhat**
A simple template for initialising a new project to be used as a data playground / data exploration project with the basics in place.

### CODE REVIEW

File: examples/data-solutions/data-playground/variables.tf [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  type        = string
+}
+
+variable "location" {
+  description = "The location where resources will be deployed."
+  type        = string
+  default     = "europe"
+}
+
+variable "project_name" {
```
Comment (ludoo) [inline examples/data-solutions/data-playground/variables.tf:26]: We use `project_id` across Fabric, can you rename this and also:

- allow reusing an existing project
- add a `prefix` variable

File: examples/data-solutions/data-playground/variables.tf [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  type        = string
+}
+
+variable "region" {
+  description = "The region where resources will be deployed."
+  type        = string
+  default     = "europe-west1"
+}
+
+variable "zone" {
```
Comment (ludoo) [inline examples/data-solutions/data-playground/variables.tf:37]: no need for a variable for this, you can either

- have only a zone variable and derive the region from that, or
- use a fixed `b` for zone in code, users can easily change it

File: examples/data-solutions/data-playground/variables.tf [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  type        = string
+  default     = "b"
+}
+
+variable "root_node" {
+  description = "The resource name of the parent Folder or Organization. Must be of the form folders/folder_id or organizations/org_id."
+  type        = string
+}
+
+variable "vpc_ip_cidr_range" {
```
Comment (ludoo) [inline examples/data-solutions/data-playground/variables.tf:48]: can you bundle all VPC related variables in a single `vpc_config` variable of type object?

**Review by ludoo — CHANGES_REQUESTED** [review]

File: examples/data-solutions/data-playground/variables.tf (hunk 1 above)
Comment (aymanfarhat) [reply examples/data-solutions/data-playground/variables.tf]: Done.

File: examples/data-solutions/data-playground/variables.tf (hunk 2 above)
Comment (aymanfarhat) [reply examples/data-solutions/data-playground/variables.tf]: Done.

File: examples/data-solutions/data-playground/variables.tf (hunk 3 above)
Comment (aymanfarhat) [reply examples/data-solutions/data-playground/variables.tf]: Done

**Comment by ludoo** [comment]
THis is great, thanks for the changes. As a last step, can you add a simple test like for the other examples, in the `tests/examples` folder?

**Comment by aymanfarhat** [comment]
@ludoo Thanks for the feedback! I added a test folder as requested and addressed the comments mentioned by @lcaggio as well (reviewed them in a 1:1).

**Comment by ludoo** [comment]
Thanks for adding tests and implementing changes! The only things left before this can be merged are

- a simple diagram that is inline with the style we use in the other examples
- alphabetical ordering of variables :)

**Review by ludoo — APPROVED** [review]

**Comment by aymanfarhat** [comment]
Hello! I hope all is well. Just checking-in regarding this PR, is there anything else needed from my end? I see that it's approved but seems to be pending a merge. Thanks!

**Pull request merged**